src/pages/Trace/FinalReview/FinalReview.tsx

```diff
@@ -3,7 +3,10 @@ import { App } from "canvas/AppController";
 
 export const FinalReview: React.FC = () => {
     useEffect(() => {
-        App.renderer.finalReviewInit();
+        //App.renderer.finalReviewInit();
```

Comment:
Yes. Since we're not showing the canvas on FinalReview page, we don't need any initialization. I removed this line and deleted the finalReviewInit() method from the renderer.